http/http_connection.hpp

```diff
@@ -443,7 +443,7 @@ class Connection :
             (req->method() == boost::beast::http::verb::patch) ||
             (req->method() == boost::beast::http::verb::delete_))
         {
-            if (res.result() == boost::beast::http::status::ok)
+            if ((res.resultInt() >= 200) && (res.resultInt() < 300))
```

Comment:
A comment why here, might make this more readable. 

//  Look for good return codes and if so we know the operation passed 
            if ((res.resultInt() >= 200) && (res.resultInt() < 300))